<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C5FilterAutoinputFlash</class>
 <widget class="QWidget" name="C5FilterAutoinputFlash">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="WdtContainer" name="wc" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2">
       <widget class="C5LineEdit" name="leStationName">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="lvalue" stdset="0">
         <string>station</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="C5DateEdit" name="leDate2">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="FilterName" stdset="0">
         <string>dateissue2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="C5LineEdit" name="leState">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="FilterName" stdset="0">
         <string>autoinput_state</string>
        </property>
        <property name="lkey" stdset="0">
         <string>autoinput_state</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="C5LineEdit" name="leStateName">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="lvalue" stdset="0">
         <string>autoinput_state</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>State</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lbIssue">
        <property name="text">
         <string>Issue date</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="C5DateEdit" name="leDate1">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="FilterName" stdset="0">
         <string>dateissue1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="C5LineEdit" name="leStation">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="FilterName" stdset="0">
         <string>station</string>
        </property>
        <property name="lkey" stdset="0">
         <string>station</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>352</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Station</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>C5LineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">c5lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>WdtContainer</class>
   <extends>QWidget</extends>
   <header location="global">wdtcontainer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>C5DateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">c5dateedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leDate1</tabstop>
  <tabstop>leDate2</tabstop>
  <tabstop>leStation</tabstop>
  <tabstop>leState</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
